### DISCUSSION TITLE
How to localize the date axis format

### DISCUSSION THREAD

**Discussion by Kiss-World** [observablehq/plot#2104] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/c390f577-b17d-4d54-a7ce-ee8025ccff8a)

Please look at the third graph, it doesn't display anything, but the first one does. I also couldn't achieve the correct date format in a regular table, maybe because I'm new to this and don't know the proper usage. However, I couldn't find more detailed instructions in the documentation

**Comment by Fil** [comment]
[formatMonth](https://observablehq.com/plot/features/formats#formatMonth) is used for month numbers (from 0 to encode January, to 11 to encode December). It does not apply to [JavaScript Dates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).

What are you trying to achieve?

**Reply by Kiss-World** [reply]
> What are you trying to achieve?

@Fil 
![image](https://github.com/user-attachments/assets/a439b852-67b7-4bbf-9fc8-1d0829dbe835)
I want the X-axis months to show Chinese months, or numeric months, as in the example below, but I won't adjust this style in a normal chart
![image](https://github.com/user-attachments/assets/fd7438e4-34ce-458d-bda7-9a2d71616511)

**Comment by Fil** [comment, marked as answer]
You could try this: https://observablehq.com/@recifs/plot-a-custom-format-for-the-date-axis

replacing `es-AR` by `zh-CN` gets you half of the way. I'm not sure how to generate the digits

**Reply by Kiss-World** [reply]
Nice!!! thanks
![image](https://github.com/user-attachments/assets/eff80c1b-79ae-4a7f-99ed-b7df4b6d3ed9)
